src/conf/data/defaults.yaml

```diff
@@ -6,13 +6,13 @@ data:
   data_sample_rate: 44100
   train_dataset:
     _target_: data.ClassifierDataset
-    data_path: '../tests/assets/data/'
+    data_path: '../datasets/recordings_2018/'
```

Comment:
Do we want to change the default file? I think we should have another configuration for running the deep voice dataset.